diff --git a/src/SharedSlice.cpp b/src/SharedSlice.cpp
index e2ce4df..00a2cb1 100644
--- a/src/SharedSlice.cpp
+++ b/src/SharedSlice.cpp
@@ -300,10 +300,6 @@ ValueLength SharedSlice::getStringLength() const {
 
 std::string SharedSlice::copyString() const { return slice().copyString(); }
 
-[[deprecated("use stringView")]] StringRef SharedSlice::stringRef() const {
-  return slice().stringRef();
-}
-
 std::string_view SharedSlice::stringView() const {
   return slice().stringView();
 }
